docs/general/kusama/kusama-getting-started.md

```diff
@@ -19,8 +19,8 @@ Kusama is owned by those who hold the Kusama tokens (KSM). There's no central ki
 changes are made through the protocol's on-chain governance.
 
 The network is a permissionless and anyone can come along and start using it. Those who participated
-in the Polkadot sale can claim a proportional amount of KSM through the [Kusama Claims
-process][claims].
```

Comment:
Here is one of many examples where prettier was splitting/breaking links

Reply:
![image](https://user-images.githubusercontent.com/13341935/208778749-f9229f08-c5a0-4265-ab26-e1f3ba0600d1.png)